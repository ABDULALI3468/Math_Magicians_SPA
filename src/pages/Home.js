/* eslint max-len: ["error", { "code": 1000 }] */
import React from 'react';
import '../components/Navbar/Navbar.css';

const Home = () => (
  <div className="home">
    <h1>Welcome to our page</h1>
    <p>
      Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quis molestiae aut, quam fugit excepturi odio vero reiciendis. Officiis atque corporis ipsum ad incidunt odit, a fugit rerum repellendus voluptatem autem dicta esse quibusdam qui, et voluptas. Provident fugit libero dolores officiis, praesentium laudantium consequatur optio accusantium asperiores tempore. Nihil quo nisi sequi autem qui repellat itaque, incidunt ullam. Temporibus laudantium nulla exercitationem id nemo officia, nobis
      maxime quos,
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quis molestiae aut, quam fugit excepturi odio vero reiciendis. Officiis atque corporis ipsum ad incidunt odit, a fugit rerum repellendus voluptatem autem dicta esse quibusdam qui, et voluptas. Provident fugit libero dolores officiis, praesentium laudantium consequatur optio accusantium asperiores tempore. Nihil quo nisi sequi autem qui repellat itaque, incidunt ullam. Temporibus laudantium nulla exercitationem id nemo officia, nobis
      maxime quos,
    </p>
  </div>
);

export default Home;
