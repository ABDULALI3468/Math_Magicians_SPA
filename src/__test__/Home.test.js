import React from "react";
import renderer from "react-test-renderer";
import Home from "../pages/Home";
import "@testing-library/jest-dom";

it("Test the Home page", () => {
  const home = renderer.create(<Home />).toJSON();
  expect(home).toMatchSnapshot();
});
