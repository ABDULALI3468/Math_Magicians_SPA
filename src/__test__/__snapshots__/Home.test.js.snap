// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Home page 1`] = `
<div
  className="home"
>
  <h1>
    Welcome to our page
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quis molestiae aut, quam fugit excepturi odio vero reiciendis. Officiis atque corporis ipsum ad incidunt odit, a fugit rerum repellendus voluptatem autem dicta esse quibusdam qui, et voluptas. Provident fugit libero dolores officiis, praesentium laudantium consequatur optio accusantium asperiores tempore. Nihil quo nisi sequi autem qui repellat itaque, incidunt ullam. Temporibus laudantium nulla exercitationem id nemo officia, nobis maxime quos,
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quis molestiae aut, quam fugit excepturi odio vero reiciendis. Officiis atque corporis ipsum ad incidunt odit, a fugit rerum repellendus voluptatem autem dicta esse quibusdam qui, et voluptas. Provident fugit libero dolores officiis, praesentium laudantium consequatur optio accusantium asperiores tempore. Nihil quo nisi sequi autem qui repellat itaque, incidunt ullam. Temporibus laudantium nulla exercitationem id nemo officia, nobis maxime quos,
  </p>
</div>
`;
